samples/Sentry.Samples.Uwp/Action.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Sentry.Protocol;
+using Windows.Storage.Streams;
+using Windows.UI;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Sentry.Samples.Uwp
+{
+    internal static class Action
+    {
+        internal static async Task TurnOnCooker()
+        {
+            var topgasAnimation = MainPage.Compositor.CreateColorKeyFrameAnimation();
+            topgasAnimation.Duration = TimeSpan.FromSeconds(5);
+            topgasAnimation.InsertKeyFrame(1.0f, Color.FromArgb(255, 51, 75, 59));
+
+            var bottomgasAnimation = MainPage.Compositor.CreateColorKeyFrameAnimation();
+            bottomgasAnimation.Duration = TimeSpan.FromSeconds(5);
+            bottomgasAnimation.InsertKeyFrame(1.0f, Color.FromArgb(255, 0, 63, 0));
+
+            MainPage.TopGradientStop.StartAnimation(nameof(MainPage.TopGradientStop.Color), topgasAnimation);
+            MainPage.BottomGradientStop.StartAnimation(nameof(MainPage.BottomGradientStop.Color), bottomgasAnimation);
+
+            SentrySdk.AddBreadcrumb("Gas leak detected", "warning", level: BreadcrumbLevel.Warning);
+            await GasLeak();
+        }
+
+        internal static async Task GasLeak()
+        {
+                MediaElement mediaElement = null;
+                var tuple = await LoadStreamAsync();
+                await UiThreadHelper.RunAsync(CoreDispatcherPriority.High, () =>
+                {
+                    mediaElement = new MediaElement();
+                    mediaElement.AutoPlay = false;
+                    mediaElement.SetSource(tuple.Item1, tuple.Item2);
+                });
+
+                await Task.Delay(3000);
```

Comment:
3 seconds for the demo is a lot, user will think it's not working

```suggestion
                await Task.Delay(1000);
```